scan/spec/test_command_generator_spec.rb

```diff
@@ -59,9 +59,11 @@
     Apple Watch - 42mm (A6371161-FEEA-46E2-9382-0DB41C85FA70) (Shutdown)
 "
     FastlaneCore::Simulator.clear_cache
-    response = "response"
-    allow(response).to receive(:read).and_return(@valid_simulators)
-    allow(Open3).to receive(:popen3).with("xcrun simctl list devices").and_yield(nil, response, nil, nil)
+    list_device_response = "list_device response"
+    allow(list_device_response).to receive(:read).and_return(@valid_simulators)
+    allow(Open3).to receive(:popen3).with("xcrun simctl list devices").and_yield(nil, list_device_response, nil, nil)
```

Comment:
Hmm..I'd be changing Open3's all over the place, how about I revert the Open3 change here?